### DISCUSSION TITLE
What is Mapbox in 1.4 ver.?

### DISCUSSION THREAD

**Discussion by dimakaob** [BimmerGestalt/AAIdrive#573] [General]
Hello! 
Please could somebody tell me what it is and how to use it in this new 1.4 AAIDRIVE?
Thank you.
I am deeply sorry if it has been already discussed. Then just give me the way, please =)

**Comment by hufman** [comment]
The Mapbox build of AAIdrive 1.4 includes a Map tab in the phone interface. After toggling the feature on, a new icon should show up in the car's Phone Apps section. Clicking into this icon should show a navigation menu and a map. That should get you started!

**Comment by dimakaob** [comment]
Thank you for the explanation!
Got it 😀

ср, 11 мая 2022 г., 23:42 Walter Huf ***@***.***>:
